internal/state/responders.go

```diff
@@ -254,7 +255,7 @@ func (u *expunge) handle(snap *snapshot, _ StateID) ([]response.Response, respon
 	}
 
 	// When handling a CLOSE command, EXPUNGE responses are not sent.
-	if u.asClose {
```

Comment:
Information `u.asClose` is not needed in this implementation, should I remove it completely?

Reply:
I think it should be safe to do so :thinking: